tools/releaser/lib/releaser.rb

```diff
@@ -314,10 +314,11 @@ def npm_otp
       " --provenance --access public"
     end
 
-    def gem_otp
+    def gem_otp(gem_path)
       " --otp " + ykman("rubygems.org")
     rescue
-      ""
+      sh "sigstore-cli sign #{gem_path} --bundle #{gem_path}.sigstore.json"
```

Comment:
Don't we need to use `bundle exec` here? The gem ins't installed in the system, is it?